lib/gvl-tracing.rb

```diff
@@ -28,6 +28,50 @@
 require_relative "gvl_tracing/version"
 
 require "gvl_tracing_native_extension"
+require "json"
 
 module GvlTracing
+  class << self
+    @@path = "/tmp/gvl-tracing.json"
```

Comment:
```suggestion
  @path = "/tmp/gvl-tracing.json"
  class << self
```

It's better to use an instance variable, you just have to define the default outside of `class << self`.